i18n/locales/source/blockly-mooc/javalab.json

```diff
@@ -23,5 +23,6 @@
   "newFile": "New File",
   "noMainMethod": "Error: your program does not contain a main method.",
   "runtimeError": "Your code hit an exception while it was running.\n{cause}",
+  "saveToBackpack": "Save to Backpack",
```

Comment:
I'm curious, when do you need to add to this javalab.json?

Reply:
🤦‍♀️ Thankyou

Reply:
(removed)